src/lib/cht-api.ts

```diff
@@ -50,10 +50,15 @@ export class ChtApi {
   public static async createSession(authInfo: AuthenticationInfo, username : string, password: string): Promise<ChtSession> {
     const COUCH_AUTH_COOKIE_NAME = 'AuthSession=';
     const protocol = authInfo.useHttp ? 'http' : 'https';
-    const sessionUrl = `${protocol}://${username}:${password}@${authInfo.domain}/_session`;
+    const sessionUrl = `${protocol}://${authInfo.domain}/_session`;
     const resp = await axios.post(sessionUrl, {
       name: username,
       password,
+    }, {
+      auth: {
+        username:username,
```

Comment:
can you use shorthand? 
```suggestion
        username,
```